src/main/java/org/team1540/base/ChickenSubsystem.java

```diff
@@ -11,14 +23,57 @@ public ChickenSubsystem(String name) {
   public ChickenSubsystem() {
   }
 
-  protected abstract double getCurrent();
+  protected void initDefaultCommand() {
+
+  }
 
   /**
-   * Set a power limit for this subsystem.
+   * Compare two ChickenCommands by priority.
+   *
+   * @param o ChickenCommand to compare to.
    *
-   * @param limit The power limit in amps.
+   * @return priority.compareTo(o.priority)
    */
-  protected abstract void limitPower(double limit);
+  public int compareTo(ChickenSubsystem o) {
```

Comment:
Why do you need this? `ChickenSubsystem` implements `PowerManageable` so the other `compareTo()` on line 76 will work fine.